recipes/gperf/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "3.0.4" %}
+{% set sha256 = "767112a204407e62dbc3106647cf839ed544f3cf5d0f0523aaa2508623aad63e" %}
+
+package:
+  name: gperf
+  version: {{ version }}
+
+source:
+  fn: gperf-{{ version }}.tar.gz
+  url: http://ftp.gnu.org/pub/gnu/gperf/gperf-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - sed  # [osx]
```

Comment:
Unfortunately, the gperf tests fail using OSX's BSD sed with:

```
../src/gperf -C -E -G -I -t < ./languages.gperf | LC_ALL=C tr -d '\r' | sed -e 's|[^ ]*[/\\][\\]*src[/\\][\\]*gperf[^ ]*|../src/gperf|' > languages.out
sed: RE error: illegal byte sequence
make[1]: *** [check-test] Error 1
make: *** [check] Error 2
```
Using gnu sed, the tests pass.